### ISSUE TITLE
Contribution of core facility to scientific discovery

### ISSUE THREAD

**Issue by ChrisVFerg** [datacite/freya#33]
As a core facility provider, I want to track usage of my facility so that I can demonstrate it's value.

**Comment by mfenner** [comment]
@ChrisVFerg What could be the proxy for usage? Number of datasets (or samples, etc.) generated? Or do you mean reuse of those datasets, e.g. in the form of usage stats?

**Comment by ChrisVFerg** [comment]
@mfenner . 'Usage' might be the wrong word here - let me rephrase to clarify things: the core facility provider wants to understand how their core facility contributes to scientific discovery: so yes - s/he wants to know what datasets were generated using the facility. But so much more than this - ie what scientific assertions and conclusions could be made (& published), what disease might have been alleviated by the discovery, how much time did the facility save researchers, etc.

**Comment by TomDemeranville** [comment]
this is very similar to one I was about to add from our user facilities group:
"As an infrastructure provider I want to be able to track outputs associated with my instruments, equipment and services so I can demonstrate the value they provide."

It's also in the PID Instruments RDA group: https://docs.google.com/document/d/1M1y1xeWYokOqUxFdNvJ0tX2wueVVcBNmz3mXU3qmZ5Q/edit?ts=5b6d6e36

**Comment by vasilyb** [comment]
This is related to #27 and #68
